src/hooks/useFrameworkTitle.ts

```diff
@@ -0,0 +1,25 @@
+import { useT } from "@transifex/react";
+
+import { Framework, useFrameworkContext } from "@/context/framework.provider";
+
+export const useFrameworkTitle = () => {
+  const t = useT();
+  const { framework } = useFrameworkContext();
+
+  switch (framework) {
+    case Framework.PPC:
+      return t("Priceless Planet Coalition");
+
+    case Framework.HBF:
+      return t("Harit Bharat Fund");
+
+    case Framework.TF:
+    case Framework.TF_LANDSCAPES:
+    case Framework.ENTERPRISES:
+      return t("TerraFund");
+
+    case Framework.UNDEFINED:
+    default:
+      return null;
+  }
+};
```

Comment:
This hook should be used in all cases where we want to display the current framework name, instead of redefining it in lots of places.